Backpack/build.gradle

```diff
@@ -59,27 +57,4 @@ dependencies {
   lintPublish project(':backpack-lint')
 }
 
-import org.jetbrains.dokka.gradle.DokkaTask
-
-task generateDokkaDocs(type: DokkaTask) {
-  def version = rootProject.ext.backpackAndroidVersion
-
-  configure {
-    outputDirectory = new File(rootProject.rootDir, "$rootProject.ext.dokkaOutputDir/versions/$version")
-
-    dokkaSourceSets {
-      named("main") {
-        moduleName = rootProject.name
-        reportUndocumented = false
-        sourceLink {
-          dir = "${project.projectDir}/src/main/java"
-          remoteUrl = new URL("https://github.com/Skyscanner/backpack-android/tree/main/${project.name}/src/main/java")
```

Comment:
I couldn't see anywhere on the dokka site where this was actually linked to. As we've got the different source sets now (e.g. java vs kotlin) this gets a lot more difficult, so I've removed this